sections/04_study.md

```diff
@@ -269,7 +270,7 @@ poorly characterized.
 Recognizing enhancers presents additional challenges. Enhancers may be up to one
 million base pairs upstream or downstream from the target promoter, on either
```

Comment:
Instead of "target", maybe "affected"? Even though I wrote the original word ...